### PR TITLE
geocouch: fix brew audit warning

### PR DESCRIPTION
**Pull request by DrAwkward2592**
@mikemcquaid I ran brew audit, and fixed a warning in geocouch. Please let me know if I did this right.

### CODE REVIEW

**Comment by DomT4** [comment]
It looks correct. All of the `(share/"geocouch")` can just be `pkgshare` as well, if you're feeling like cleaning up the formula :smiley:. That should be highlighted by `brew audit --strict`.

Maintainer note: If this passes CI it needs new bottles.

**Comment by mikemcquaid** [comment]
> It looks correct. All of the (share/"geocouch") can just be pkgshare as well, if you're feeling like cleaning up the formula .

Yeh, that'd be great. It's also worth reading `.github/CONTRIBUTING.md` and fixing up the commit message. Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

**Comment by mikemcquaid** [comment]
It looks like the `sha256` of the download may also need updating: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/43616/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/fetch_geocouch/

**Comment by DomT4** [comment]
Oh, the build failure is [this old fun issue](https://github.com/Homebrew/homebrew/issues/42498).

**Comment by DrAwkward2592** [comment]
I fixed the git commit message, and changed a few (share/"geocouch") to pkgshare. I am not sure how to update sha256.

File: Library/Formula/geocouch.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def install
     (share/"geocouch/geocouch.plist").chmod 0644
 
     #  Install geocouch.ini into couchdb.
-    (etc/"couchdb/default.d").install Dir["etc/couchdb/default.d/geocouch.ini"]
+    (etc/"couchdb/default.d").install ["etc/couchdb/default.d/geocouch.ini"]
```
Comment (mikemcquaid) [inline Library/Formula/geocouch.rb]: You can actually strip the `[` and `]` altogether.

File: Library/Formula/geocouch.rb [hunk 2]
```diff
@@ -37,12 +37,12 @@ def install
...
     #  Install geocouch build files.
-    (share/"geocouch").mkpath
+    pkgshare.mkpath
     rm_rf share/"geocouch/ebin/"
-    (share/"geocouch").install Dir["ebin"]
+    pkgshare.install "ebin"
 
     #  Install geocouch.plist for launchctl support.
-    (share/"geocouch").install Dir[couchdb_dir/"etc/launchd/org.apache.couchdb.plist.tpl.in"]
+    pkgshare.install Dir[couchdb_dir/"etc/launchd/org.apache.couchdb.plist.tpl.in"]
```
Comment (mikemcquaid) [inline Library/Formula/geocouch.rb]: You can actually strip the `[` and `]` altogether.

File: Library/Formula/geocouch.rb [hunk 3]
```diff
@@ -18,11 +18,11 @@ class Geocouch < Formula
   depends_on "erlang" => :build
 
   def couchdb_share
-    HOMEBREW_PREFIX/"share/couchdb"
+    HOMEBREW_PREFIX/pkgshare
   end
 
   def geocouch_share
```
Comment (mikemcquaid) [inline Library/Formula/geocouch.rb]: Pretty sure this function can be removed and all callers replaced with `pkgshare`

File: Library/Formula/geocouch.rb [hunk 4]
```diff
@@ -18,11 +18,11 @@ class Geocouch < Formula
   depends_on "erlang" => :build
 
   def couchdb_share
-    HOMEBREW_PREFIX/"share/couchdb"
+    HOMEBREW_PREFIX/pkgshare
```
Comment (mikemcquaid) [inline Library/Formula/geocouch.rb]: This is not correct so should just stay as it was.

**Comment by mikemcquaid** [comment]
@DomT4 You OK to pull this as-is when the comments are addressed?

**Comment by mikemcquaid** [comment]
This may also need rebased on `master`. Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

**Comment by DomT4** [comment]
@mikemcquaid Yeah. I'll need to look at the sandbox situation here when I get a chance, but this is fine to go in without a bottle bump till then.

**Comment by mikemcquaid** [comment]
@DomT4 👍

File: Library/Formula/geocouch.rb [hunk 5]
```diff
@@ -22,7 +22,7 @@ def couchdb_share
   end
 
   def geocouch_share
-    HOMEBREW_PREFIX/"share/geocouch"
+    HOMEBREW_PREFIX/pkgshare
```
Comment (DomT4) [inline Library/Formula/geocouch.rb]: Fixed this in post but this one was fine not to change FWIW :smiley:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/36c64e6e57439538fb6796c1979b1c5437f620af. Thank you for your contribution to Homebrew @DrAwkward2592; we appreciate it! :smiley_cat:

**Pull request closed without merging**